pmm/conan.cmake

```diff
@@ -280,6 +280,7 @@ function(_pmm_conan_get_settings out)
     if(NOT ARG_SETTINGS MATCHES ";?os=")
         _pmm_log(DEBUG "Using os=${os}")
         list(APPEND ret os=${os})
+        list(APPEND ret os_build=${os})
```

Comment:
See #15 . It's trickier than just grabbing the OS and architecture, as CMake can do cross compilation. It needs the CMake HOST variables.

Reply:
I know, but this was more intended to be a quick fix because at the moment some packages fail to build